IPy.py

```diff
@@ -1383,6 +1383,8 @@ def parseAddress(ipstr):
     >>> testParseAddress('::FFFF:129.144.52.38')
     281472855454758 (IPv6)
     """
+    """ Make sure there is no space in `ipstr`, if there is raise `ValueError`."""
+    if " " in ipstr: raise ValueError("IP Address format was invalid: %s" % ipstr)
```

Comment:
I did not consider to check for `\t` because it shows apparent difference.

This error happened when I do user test. I asked some people to connect to a database using interactive CLI. I saw some users accidentally add additional space(s) between the dot. Furthermore, in some terminal emulator, it is hard to see if there is a space or not.